docs/dbtoken-service.md

```diff
@@ -8,12 +8,13 @@ In brief, there are three options:
 
 1. use r/w or r/o primary keys, which grant access to the whole database account
 2. implement a service which transforms (1) into scoped resource tokens
-3. a third AAD RBAC-based model is in preview.
+3. a third AAD RBAC-based model with Entra Ids.
```

Comment:
What did you mean here by "with Entra Ids"?

I would have written this line as:

```
3. a third Microsoft Entra ID RBAC-based model
```

or maybe to make it a bit less formal:

```
3. a third Azure RBAC-based model
```

Reply:
Can address as part of follow-up as it's documentation.

Reply:
That's fair. PR here: https://github.com/Azure/ARO-RP/pull/3480